src/components/header/Header.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import '../body/RoutesApp/RoutesApp.css';
+
+function Header() {
+    const navStyle = {
+        color: 'white'
+    };
+    return (
+        <nav className="navbar navbar-expand-lg navbar-light bg-light">
+            <a className="navbar-brand" href="#">Aruba HTS</a>
+            <button className="navbar-toggler" type="button" data-toggle="collapse"
```

Comment:
whats this button for?

Reply:
In mobile view, this is a hamburger, which will open the menu
so basically a menu collapses into this in smaller screens